Namespace(batch_size=32, data_dir='/root/dataset/Osteoporosis/Sagittal_Contrast_Final_1/', epochs=60, log='logs/Ordinal', lr=0.0001, n_gpu=4, num_classes=3, phase='train', seed=0, weight_decay=0.001, workers=16)
ordinal.py:41: UserWarning: You have chosen to seed training. This will turn on the CUDNN deterministic setting, which can slow down your training considerably! You may see unexpected behavior when restarting from checkpoints.
  warnings.warn('You have chosen to seed training. '
Traceback (most recent call last):
  File "ordinal.py", line 251, in <module>
    main(args)
  File "ordinal.py", line 96, in main
    train(model, train_loader, criterion, optimizer,
  File "ordinal.py", line 148, in train
    acc = ordinal_accuracy(output, target)
NameError: name 'ordinal_accuracy' is not defined
